src/NavBarFunc2.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Why is there a `NavBarFunc2.tsx`? It looks like it's the same as `NavBarFunc2.tsx` - were you possibly using this to test what would fix the lint errors and forgot to delete it?

Reply:
Yes, I will remove this.